### PR TITLE
Add expiry date checking

### PR DESCRIPTION
**Pull request by tadityar**
PR for issue #2 . verify.js was generated by running `npm run build` but seeing the large diff from the file I am not sure whether I did it the correct way. Currently there's no test here, I can verify that the valid cert will fail if I add `"expires" : "<some date in the past>"` but I don't know how to correctly generate a new certificate JSON. Should I issue it using the cert-issuer? Copy-pasting the valid certificate and adding `expires` field makes the hash comparison fails.

Thanks for making blockcerts!

### CODE REVIEW

**Comment by kimdhamilton** [comment]
Thank you Tasya! I have an expired certificate that we can use as a test. I will add this to your branch, run the tests, and merge it into master. 

Thanks so much for taking the time to do this!

**Comment by kimdhamilton** [comment]
about the large verifier.js diff -- I'll rerun to ensure it's ok

**Comment by kimdhamilton** [comment]
I couldn't find issued certs that were expired, so I added targeted tests and it looks good!

The unexpected verifier diffs were from another change, so it wasn't your change that caused it. Everything looks good -- thanks again!

File: lib/verifier.js [hunk 1]
```diff
@@ -467,9 +467,25 @@ export class CertificateVerifier {
...
+      // expiry date not set, nothing to do
       return this._succeed(completionCallback);
     }
 
+    var expiryDate = Date.parse(this.certificate.expires);
+
+    if (new Date() >= expiryDate) {
+      var reason = "This certificate has expired.";
+      return this._failed(completionCallback, reason, null);
+    }
```
Comment (tadityar) [inline lib/verifier.js]: Thanks for reviewing my PR! I had this uneasy feeling about something and turns out I missed putting `return this._succeed(completionCallback);` at the end of the function here. Sorry for the trouble.

Comment (kimdhamilton) [inline lib/verifier.js]: glad you noticed this! I'll add a targeted test for that too

**Pull request merged**